test/util/validator.test.js

```diff
@@ -1,7 +1,7 @@
 const { validateInputJson } = require('../../src/util/validator')
 
-test('validateInputJson', () => {
-  const goodRequest = {
+describe('validateInputJson', () => {
+  const request = {
```

Comment:
Move to 'Good request' test

Reply:
The enum object is used by multiple tests.

Reply:
The request object is used by almost all tests.